util/resources.go

```diff
@@ -56,14 +61,11 @@ func memconv(mem string) (int64, error) {
 		multiplier = 1000000000000
 	}
 
-	if multiplier == -1 {
```

Comment:
This no longer guards against given input without a unit, which should default to bytes.